features/record_of_guidance.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Record of guidance
```

Comment:
:+1: 

Can we also include a scenario(s) to cover the additional information we're collecting but not including in the document. Off the top of my head I can think of:

- `income_in_retirement`
- `value_of_pension_pots`
- `upper_value_of_pension_pots`
- `value_of_pension_pots_is_approximate`